src/client/Microsoft.Identity.Client.Broker/Microsoft.Identity.Client.Broker.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworkNetStandard>netstandard2.0</TargetFrameworkNetStandard>
     <TargetFrameworkNetDesktop Condition="$([MSBuild]::IsOsPlatform('Windows')) and '$(MSAL_DESKTOP_ONLY_DEV)' == ''">net461</TargetFrameworkNetDesktop>
 
+    <LangVersion>8.0</LangVersion>
```

Comment:
Inconsistent with MSAL? Why is this here?

Reply:
8.0 is required to declare using of a local variable. e.g.
``` CSharp
using LogEventWrapper logEventWrapper = new LogEventWrapper(this);
```